common/faceting.js

```diff
@@ -59,6 +59,7 @@
                         ctrl.facetingCount++;
 
                         $scope.vm.facetModels[index] = {
+                            facetError: false,
```

Comment:
Just for the sake of bookkeeping, can you please add the `noConstraints:false` to here too. So we would know that the facetModel can have such attribute.